contributed_definitions/nyaml/NXelectronanalyser.yaml

```diff
@@ -49,26 +49,35 @@ NXelectronanalyser(NXobject):
       As a result, the measured kinetic energy :math:`E_K^{\mathrm{meas.}}` of a photoelectron is `independent`
       of the sample work function. Nonetheless, the work function :math:`\phi_s` needs to be known to
       accurately determine the binding energy scale.
-  energy_resolution(NX_FLOAT):
-    unit: NX_ENERGY
+  energy_resolution(NXresolution):
     doc:
     - |
-      Energy resolution of the electron analyser (FWHM of gaussian broadening).
+      Energy resolution of the analyser with the current setting. May be linked from a
+      NXcalibration.
     - |
       xref:
-        spec: 18115-1:2023
-        term: 10.7
-        url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:10.7
-  momentum_resolution(NX_FLOAT):
-    unit: NX_WAVENUMBER
+        spec: ISO 18115-1:2023
+        term: 10.24
+        url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:10.24
+    physical_quantity:
+      enumeration: [energy]
+    resolution(NXdata):
```

Comment:
This is NX_FLOAT in the base class (also makes more sense)

Reply:
This was exactly my point about the error bars raised [here](https://github.com/FAIRmat-NFDI/nexus_definitions/pull/52#discussion_r1338268682). If resolution is NX_FLOAT, then there is no way to talk about its standard deviation.

Reply:
We could either make `resolution` of type `NXdata` OR keep `resolution(NX_FLOAT)` and add `resolution_errors(NX_FLOAT)`.

Reply:
I think both are fine. Using `NXdata`, however, feels a bit more aligned with our general movement towards using `NXdata` to represent physical quantities.

Reply:
Agreed. If @rettigl agrees, I would change them all to `NXdata`, also in the base class.

Reply:
Used NX_FLOAT and introduced resolution_error(NX_FLOAT)